src/Microsoft.Azure.SignalR.Common/ServiceConnections/ClientConnectionScope.cs

```diff
@@ -67,7 +68,7 @@ internal static IServiceConnection OutboundServiceConnection
 
         internal static bool IsDiagnosticClient
         {
-            get => ScopePropertiesAccessor<ClientConnectionScopeProperties>.Current?.Properties?.IsDiagnosticClient ?? false;
+            get => ScopePropertiesAccessor<ClientConnectionScopeProperties>.Current.Properties?.IsDiagnosticClient ?? false;
```

Comment:
`ScopePropertiesAccessor<ClientConnectionScopeProperties>.Current` never be null?